.github/CODEOWNERS

```diff
@@ -7,20 +7,16 @@
 # be requested for review when someone opens a pull request.
 # order is alphabetical for easier maintenance.
 #
-# AnandatDell (anandrajak1)
+# Aaron Tye (atye)
+# Aly Nathoo (anathoodell)
 # Chiman Jain (chimanjain)
-# Don Khan (donatwork)
+# Evgeny Uglov (EvgenyUglov)
 # Harish H  (HarishH-DELL)
-# Harsha Yalamanchili (harshaatdell)
 # Nitesh Rewatkar (nitesh3108)
-# Prabhu Revur (prabhu-dell)
-# Peter Cao (P-Cao)
+# Oleksandr babiychuk (alexemc)
```

Comment:
Last name should start with a capital letter.

Reply:
Fixed